compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/PlatformContext.skiko.kt

```diff
@@ -123,6 +123,8 @@ interface PlatformContext {
 
     val dragAndDropManager: PlatformDragAndDropManager get() = EmptyDragAndDropManager
 
+    val keepScreenOnManager: PlatformKeepScreenOnManager get() = EmptyKeepScreenOnManager
+
```

Comment:
I don't think that we need to expose this counter to platform ("semi" public API)

WDYT about
```suggestion
    var isKeepScreenOn: Boolean = false
```
?

Reply:
Yes, agree that it would be better that platform isn't exposed to the counter. I wasn't sure whether there would be different approaches to the counter for different platforms. But I have updated the implementation to use the single flag.